run/run.py

```diff
@@ -254,6 +244,29 @@ def main(platform_id, platform, args, config):
     print('Response text:', response.text)
 
 
+
+def setup_wpt(mainargs, platform, config, logger):
```

Comment:
Can you add some PEP484 type annotations?

Reply:
I can, but maybe that should occur in a follow up PR? There are no such annotations elsewhere in this file, but it would be nice to add them for _all_ functions in this file.